courses/beginner/10.md

```diff
@@ -48,207 +41,181 @@ control 'nginx-modules' do
-
- - nginx-version
- - nginx-modules
- - nginx-conf
- - first-overlay-control
- - third-overlay-control
 
-Controls `second-overlay-control`, `fourth-overlay-control`, and `fifth-overlay-control` would not be run, just as if they were manually skipped. This method of including specific controls ensures only the controls specified are executed; if new controls are added to a later version of `my_nginx_overlay`, they would not be run.
+Whenever `my_nginx_overlay` is executed, it will run only the controls from `my_nginx` that are specified in the `require_controls` block. In the case, the following controls would be executed:
 
-And, just the way its possible to modify controls when using `include_controls`, controls can be modified as well.
+| Controls      |     Executed  |
+| ------------- | ------------- |
+| nginx-version |      &check;  |
+| nginx-modules |      &check;  |
+| nginx-conf-file |     &cross;  |
+| nginx-conf-perms |   &cross;   |
+| nginx-shell-access |  &cross;  |
 
+Controls `nginx-conf-file`, `nginx-conf-perms`, and `nginx-shell-access` would not be executed, just as if they were manually skipped. This method of including specific controls ensures only the controls specified are executed
 
-```ruby
-# my_nginx profile  
-control 'nginx-version' do
-  impact 1.0
-  title 'NGINX version'
-  desc 'The required version of NGINX should be installed.'
-end
+::: warning
+If new controls are added to a later version of `my_nginx`, they would not be executed unless explicitly required in this scenario.
+:::
 
-control 'nginx-modules' do
-  impact 1.0
-  title 'NGINX modules'
-  desc 'The required NGINX modules should be installed.'
-end
+### Modifying a Control  
 
-control 'nginx-conf' do
-  impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
-end
+Let’s say a particular control from an included profile should still run, but the impact isn’t appropriate? When a control is included or required, it can also be modified!
 
-require_controls 'my_nginx_overlay' do
-  control 'first-overlay-control' do
+::: code-tabs
+@tab include_controls
+```ruby
+include_controls 'my_nginx' do
+  control 'nginx-modules' do
     impact 0.5
   end
-  control 'third-overlay-control'
 end
 ```
 
+@tab require_controls
 ```ruby
-# my_nginx_overlay
-control 'first-overlay-control' do
-  impact 1.0
+require_controls 'my_nginx' do
+  control 'nginx-modules' do
+    impact 0.5
+  end
+  control 'nginx-conf-file'
 end
+```
+
+:::
+
+In the above example, all included or required controls from `my_nginx` profile are executed. However, should control `nginx-modules` fail, it will be raised with an impact of `0.5` instead of the originally-intended impact of `1.0`.
 
-control 'second-overlay-control'
-control 'third-overlay-control'
-control 'fourth-overlay-control'
-control 'fifth-overlay-control'
+#### Inputs with profile inheritance
```

Comment:
#### Combining Inputs with profile dependencies to further enhance Overlay profiles